### ISSUE TITLE
Bump jQuery version from 2 to 3

### ISSUE THREAD

**Issue by elisechant** [Shopify/slate#431]
### Problem

Currently the jQuery version in Slate is 2.2.3, bump it to latest jQuery which is now 3.3.1. I need to down port my custom jQuery scripts to support the lower version. 

### Replication steps

n/a

### More Information

`/scripts/vendor/jquery-2.2.3.min.js`

**Comment by NathanPJF** [comment]
@elisechant Is this only a suggestion to have [the current `slate-theme` in Slate v0](https://github.com/Shopify/slate/tree/master/packages/slate-theme/src/scripts/vendor) come with a higher version of jQuery? Or are you also/instead saying there's an issue you're experiencing with replacing this vendor script with the one you would like to use?

You should be able to swap out with which ever vendor scripts you want, but I saw your comment here:

>  I need to down port my custom jQuery scripts to support the lower version.

Which makes me think there's a problem with switching.

**Comment by elisechant** [comment]
Yes, whatever the default theme is. I thought v2 seemed a bit low. Not having user issues.
